docs/management/connectors/action-types/inference.asciidoc

```diff
@@ -59,12 +61,14 @@ The following example performs a completion task on the example question.
 Input::
 The text on which you want to perform the {infer} task. For example:
 +
+--
 [source,text]
 --
 {
   input: 'What is Elastic?'
 }
 --
+--
```

Comment:
```suggestion

Input::
The text on which you want to perform the {infer} task. For example:
+
[source,text]
--
{
  input: 'What is Elastic?'
}
--
```

I think there's a few instances of mis-formatting here?

Without the space above `Input::` it won't render properly and I there's too many `--` delimiters?